k8s/chart/values.yaml

```diff
@@ -0,0 +1,11 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicas: 1
+image:
+  registry: andrewstuart
```

Comment:
You can build and run it locally, but for distribution you really want to host it on the hub or a registry of some sort. For k8s, that's a baseline requirement as each node is transient and will need to pull the pre-built image from somewhere the first time it starts up. You definitely don't want to be building the docker image each time a node wants to run the portal for the first time, as it's a noticeably time-consuming process.